### ISSUE TITLE
Sign up not mentioning T&Cs

### ISSUE THREAD

**Issue by selfthinker** [CodePen/CodePen-Bug-Reporting-Only#235]
When you sign up to CodePen, there is nothing which says you accept the [T&Cs](https://blog.codepen.io/legal/terms-of-service/), they are not even mentioned. I am not sure this is legal. (But IANAL.)
Even afterwards they are not even linked anywhere (most websites link them in the footer), you have to google to find them.

I only registered recently and found that all pens are automatically MIT-licensed. Although I would have MIT-licensed my so far only pen anyway, I don't like that it is forced on you and am pretty sure it is illegal (or at least highly immoral) that you don't tell people about it when they sign up!
I found out only by looking at the source code of the full page of the pen and then googling and finding the [FAQ on licensing](https://blog.codepen.io/legal/licensing/). Then I signed up under a new user (which I deleted again right after) just to see where you might have mentioned any of that. But you don't!

Please at least mention the T&Cs during sign up.

**Comment by chriscoyier** [comment]
Hi Anika,

Agreed: we should make links to the [Terms](https://blog.codepen.io/legal/terms-of-service/) and [Privacy](https://blog.codepen.io/legal/privacy/) more findable from the signup page. I'll put that on my personal to-do. But I'll be closing this as it's not a bug. Those pages are part of our [documentation](https://blog.codepen.io/documentation/), along with our [Code of Conduct](https://blog.codepen.io/legal/code-conduct/) and the licencising explainer that you found.

We talked to a lawyer about whether we need to ask people to specifically agree to terms before signing up, and the legal answer is no and that isn't even particularly common anymore. I also disagree that it is "highly immoral". The license is found when downloading any public code, or in the source itself if inspected. The terms are largely a boilerplate document that apply to apply to most websites, customized slightly to be specific to CodePen, and of which we've interpreted in Plain English to make them easier to understand.

**Comment by selfthinker** [comment]
> We talked to a lawyer about whether we need to ask people to specifically agree to terms before signing up, and the legal answer is no and that isn't even particularly common anymore.

I cannot imagine that to be true. I know it is getting more common to not ask for agreement (e.g. with a checkbox) but then you would need to say something like "By signing up you agree to our (linked) terms". Otherwise you make a "contract" with someone (not sure if it's really a contract) without them being aware that there is one.

I cannot imagine how something can be legally binding if I don't even know about its existence!?

> I also disagree that it is "highly immoral". The license is found when downloading any public code

Well, let's agree to disagree then.
When the license can only be found in the code (and not even in the part of the code that people see and copy), it is hidden from view for all intends and purposes. What makes this case even worse is that it is even hidden from people who copy the code who might then **not** include the copyright or the license and will therefore violate both.
